near/omni-bridge/src/lib.rs

```diff
@@ -981,6 +960,29 @@ impl Contract {
         )
     }
 
+    #[payable]
+    #[access_control_any(roles(Role::DAO))]
+    pub fn set_utxo_chain_token(
+        &mut self,
+        chain_kind: ChainKind,
+        token_id: AccountId,
+        decimals: u8,
+    ) {
+        let storage_usage = env::storage_usage();
+        let token_address = OmniAddress::new_zero(chain_kind)
+            .unwrap_or_else(|_| env::panic_str("ERR_FAILED_TO_GET_ZERO_ADDRESS"));
+
+        self.add_token(&token_id, &token_address, decimals, decimals);
```

Comment:
Do we do a storage deposit? if not, then we have to do that to avoid configurations mistakes